src/impact-dashboard/CurveChart.js

```diff
@@ -0,0 +1,120 @@
+import { curveCatmullRom, format } from "d3";
```

Comment:
Can we use `.tsx` for new files going forward? If you want a quick intro to ts if you haven't used it before, just let me know!

Reply:
I tried, I spent some time with the introductory docs yesterday and mostly just wasted a bunch of time thrashing ... when we set this up you said I'd be able to write regular javascript, and I'm not so sure it's the best use of my time right now to stop development and try to learn enough to get by. maybe after we get v0 launched? can I just enable `allowJS` in `tsconfig` for now and move on?

Reply:
We do allow js already!

JS is fine for now. The main thing I'd like to note is that TS makes it easier for other people (including you, at a later time) to understand your code. We can definitely postpone the conversion! Maybe later this week?

Reply:
ah, I think I got my local env into a weird state where it was rejecting some of my JS imports, but yes you're right it does work as is. I'm sure TypeScript is lovely, I have been meaning to learn it someday but I think today is not that day! Let's maybe talk about it on Friday, in the meantime pls approve if everything else outstanding has been addressed (I think it has?)